### DISCUSSION TITLE
"missing" and "empty" labels

### DISCUSSION THREAD

**Discussion by Kieran31** [ultralytics/yolov5#7148] [Q&A] [answered]
Hi, 

In the Train Custom Data [Tutorial](https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data#:~:text=if%20no%20objects%20in%20image%2C%20no%20*.txt%20file%20is%20required), there is "if no objects in image, no `*.txt` file is required". I'm curious about whether these non-objects images are ignored during the training. 
My dataset has 100 images with no objects. 20 of them have corresponding `*.txt` files. So when I ran `train.py`, it shew 
`Scanning ‘../datasets/data’ images and labels... 2000 found, 80 missing, 20 empty, 0 corrupted`. How are "missing" and "empty" image label pairs treated? Are they passed into the model like normal images (expect for loss calculation with empty target)?

**Comment by glenn-jocher** [comment, marked as answer]
@Kieran31 ALL images are used during training. If no labels are found then the image simply has no labels.

'missing' labels mean that no labels.txt file is found, which is fine, the image simply has no objects. 'empty' means an empty labels.txt file was found. They both have the same result.

**Reply by Kieran31** [reply]
Thank you so much

**Reply by AkramSyed002** [reply]
hy there is there any way to force it to create empty label file for that image which is not matches?

**Comment by 27samarpan** [comment]
Sir the process of training a tif file is same as normal jpg image? I tried to do so but i am getting :  ignoring corrupt image/label: cannot identify image file
this error
Can you please help
